test/visual_regression/tests/discover/chart_visualization.js

```diff
@@ -52,18 +52,21 @@ export default function ({ getService, getPageObjects }) {
 
       it('should show bars in the correct time zone', async function () {
         await PageObjects.header.awaitGlobalLoadingIndicatorHidden();
+        await visualTesting.showElementInPercy('.echChart');
```

Comment:
I would prefer if we passed an array of test subject selectors to `await visualTesting.snapshot();`, then in the [browser logic](https://github.com/elastic/kibana/blob/master/test/visual_regression/services/visual_testing/take_percy_snapshot.js#L32) we can choose to either capture based on a whitelist or blacklist. If there is a whitelist of elements to capture then we could attach the `.hideInPercy` class to the `<body>` and the `.showInPercy` class to the whitelisted elements, then remove the attached classes, all synchronously in a single `executeScript()` call.

When the whitelist isn't defined the entire page can be captured. If we don't want that then we can just require the whitelist.

Reply:
Ok good idea, I was gonna ask if you would like the ability to choose to whitelist or blacklist.

Reply:
see 0e44af7 let me know if you were thinking of a different way of doing it than that.